spec/support/evm_spec_helper.rb

```diff
@@ -12,6 +12,15 @@
 module EvmSpecHelper
   extend RSpec::Mocks::ExampleMethods
 
+  module EmsFolder
```

Comment:
@Fryguy - Regarding changing it into a normal Factory - We (@bdunne and I) ran into a lot of issues attempting to make that work correctly. One problem in particular was an error around `NoMethodError` on  `parent=`  

FactoryGirl didn't like the call to `.tap` in order to work around that issue. 

The helper idea came out of all those problems, and it just worked.